tests/tracking/fluent/test_fluent.py

```diff
@@ -1687,3 +1687,27 @@ def test_runs_are_ended_by_run_id():
         mlflow.start_run(run_id=run.info.run_id)
 
     assert mlflow.active_run() is None
+
+
+def test_last_logged_model():
+    assert mlflow.last_logged_model() is None
+
+    m1 = mlflow.create_logged_model()
+    assert mlflow.last_logged_model().model_id == m1.model_id
+
+    m2 = mlflow.create_logged_model()
+    assert mlflow.last_logged_model().model_id == m2.model_id
+
+    client = MlflowClient()
+    client.set_logged_model_tags(m2.model_id, {"tag": "value"})
```

Comment:
Should `last_logged_model` reflect updates like setting a tag?

Reply:
No need - I don't think `last_active_run()` does this.

Reply:
It actually does. It uses `get_run` to fetch the run. Here's an example:

```python
import mlflow

with mlflow.start_run() as run:
    pass

mlflow.MlflowClient().set_tag(run.info.run_id, "key", "value")

assert "key" in mlflow.last_active_run().data.tags
```

Reply:
Nice!